### DISCUSSION TITLE
NRMSE calculation in prediction tutorial

### DISCUSSION THREAD

**Discussion by lcastri** [jakobrunge/tigramite#252] [Q&A]
Hi,

I was wondering if the calculation of the NRMSE in the prediction tutorial (https://github.com/jakobrunge/tigramite/blob/master/tutorials/tigramite_tutorial_prediction.ipynb) is correct.

you compute the NRMSE as follows:
```
np.abs(true_data - predicted).mean()/true_data.std()
```
but this seems to be the mean of the **absolute** error normalised by the true data's std.

Please correct me if I am wrong.
Thanks Luca

**Comment by jakobrunge** [comment]
Good catch, I changed the name to normalized mean absolute error (NMAE).
